src/a-d/Adl/Adl.BufferList.cs

```diff
@@ -15,6 +15,16 @@ public AdlList()
         _previousAdl = 0;
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AdlList"/> class with initial quotes.
+    /// </summary>
+    /// <param name="quotes">Initial quotes to populate the list.</param>
+    public AdlList(IReadOnlyList<IQuote> quotes)
+        : this()
```

Comment:
Does base primary ctor exec before this one?
I’m concerned that `_previousAdl` gets reset at the wrong time.  Ensure our tests for this pattern adds half the quotes on instantiation and the remainder after, then compare to series equivalent.  Many places.